idea/gearexe/frontend/src/components/header/header.module.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  position: relative;
+  width: 100vw;
+  max-width: 1440px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 48px;
+  height: 78px;
```

Comment:
If it's possible I wouldn't specify explicit height at all, otherwise it's too easy to encounter overflow issues on different font sizes or future changes in layout.